### PR TITLE
Ensure IDX doc is created and linked to DID

### PR DESCRIPTION
**Pull request by PaulLeCam**

### CODE REVIEW

**Review by oed — APPROVED** [review]
Looks good. Feel free to update the `DID` prop to `id`.

File: src/identity-wallet.ts [hunk 1]
```diff
@@ -43,10 +43,10 @@ export default class IdentityWallet {
   }
 
   /**
-   * @property {string} DID                 The 3ID of the IdentityWallet instance
+   * @property {string} id                 The 3ID of the IdentityWallet instance
```
Comment (oed) [inline src/identity-wallet.ts:46]:
```suggestion
   * @property {string} id                 The DID of the IdentityWallet instance
```

**Review by oed — CHANGES_REQUESTED** [review]
One thing is missing here. The `addAuthEntries` method must now check to make sure that the authKeychain is in the root index.

**Comment by PaulLeCam** [comment]
@oed good point, thanks for catching this!
I made the change so that the auth keychain doc is always created even if empty, this should cover the case.

**Review by oed — APPROVED** [review]
Thanks for the update. Lmk if you need a release directly!

**Comment by PaulLeCam** [comment]
Thanks! It's not a blocker for me at the moment but might be helpful for the community to have a new release.

**Pull request merged**